Example/Podfile

```diff
@@ -133,7 +133,7 @@ target 'Auth_Sample' do
   pod 'FirebaseCoreDiagnostics', :path => '../'
   pod 'FirebaseCoreDiagnosticsInterop', :path => '../'
   pod 'FBSDKLoginKit'
-  pod 'GoogleSignIn', '~> 4.4'
+  pod 'GoogleSignIn'
```

Comment:
Should it be , '~> 5.0'

Reply:
Then we won't always get the latest version right?

Reply:
Right - I suppose this is better so that we catch it when 6.0 happens :)